src/spans/sqlSpan.ts

```diff
@@ -24,13 +24,13 @@ export const createSqlSpan = (
     ...baseSpan,
     started: requestMetadata.started,
     connectionParameters: {
-      host: dbFields.connectionParameters.host,
-      port: dbFields.connectionParameters.port,
-      database: dbFields.connectionParameters.database,
-      user: dbFields.connectionParameters.user,
+      host: dbFields?.connectionParameters?.host,
+      port: dbFields?.connectionParameters?.port,
+      database: dbFields?.connectionParameters?.database,
+      user: dbFields?.connectionParameters?.user,
     },
-    query: normalizeQuery(dbFields.query),
-    values: dbFields.values ? payloadStringify(dbFields.values) : '',
+    query: normalizeQuery(dbFields?.query),
+    values: payloadStringify(dbFields?.values) || '',
```

Comment:
wouldn't `payloadStringify(dbFields?.values)` always result in a value?

Reply:
Fair